### PR TITLE
docs: ChatGPT Readme

### PR DESCRIPTION
**Pull request by eriklimakc**

### CODE REVIEW

File: UI/ChatGPT/README.md [hunk 1]
```diff
@@ -1,99 +1,30 @@
-# ChatGPT - MVUX
+# ChatGPT
 
-## Guidance to use Records with MVUX
+This is a ChatGPT Sample app using the OpenAI SDK. The app was built using C# Markup for the UI and MVUX as the design pattern, encouraging the flow of immutable data. Immutable records are used to manage state and ensure immutability.
```
[the remark this answers is not in the document]
Comment (eriklimakc) [reply UI/ChatGPT/README.md]: Fixed in next commit.

File: UI/ChatGPT/README.md [hunk 2]
```diff
@@ -1,99 +1,30 @@
-# ChatGPT - MVUX
+# ChatGPT
 
-## Guidance to use Records with MVUX
+This is a ChatGPT Sample app using the OpenAI SDK. The app was built using C# Markup for the UI and MVUX for state management, encouraging the flow of immutable data. Records are used to manage state and ensure immutability.
```
Comment (rafael-rosa-knowcode) [inline UI/ChatGPT/README.md:3]: @eriklimakc 

I would add an image of the project running here.
But it's just a suggestion. 
If you understand that it is not necessary, the content is already great.

**Review by rafael-rosa-knowcode — APPROVED** [review]

**Review by iurycarlos — APPROVED** [review]
Nice job!

File: UI/ChatGPT/README.md (hunk 2 above)
Comment (kazo0) [inline UI/ChatGPT/README.md:3]:
```suggestion
This is a ChatGPT Sample app using the OpenAI SDK. The app was built using C# Markup for the UI and MVUX for state management, encouraging the flow of immutable data. [Records](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record) are used to manage state and ensure immutability.
```

File: UI/ChatGPT/RecordsGuidance.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+# Guidance to use Records with MVUX
+
+## What is a Record
+
+A record behaves like a class, offering the feature of **immutability**, where the values assigned to it remain unchanged once set. It's possible to create records using the `record` modifier, for example:
```
Comment (kazo0) [inline UI/ChatGPT/RecordsGuidance.md:5]:
```suggestion
A [record](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record) behaves like a class, offering the feature of **immutability**, where the values assigned to it remain unchanged once set. It's possible to create records using the `record` modifier, for example:
```

**Review by kazo0 — APPROVED** [review]

File: UI/ChatGPT/README.md [hunk 4]
```diff
@@ -1,99 +1,32 @@
...
-
-//Then you can update your message list displayed in the UI, thread-safe
-await Messages.UpdateAsync(message);
-...
-
-```
+ * [**MainPage.cs**](ChatGPT/ChatGPT/Presentation/MainPage.cs): Defines the main user interface using C# Markup.
+ * [**MainModel.cs**](ChatGPT/ChatGPT/Presentation/MainModel.cs): Handle user input and methods to interact with the AI.
+ * [**ChatService.cs**](ChatGPT/ChatGPT/Services/ChatService.cs): Manages communication with the OpenAI API using the OpenAI SDK. This file contains the code responsible for making requests to the OpenAI API and handling the responses.
+ * [**Message.cs**](ChatGPT/ChatGPT/Presentation/Message.cs): Record that represents a message sent by user or the AI.
```
Comment (agneszitte) [inline UI/ChatGPT/README.md:27]:
```suggestion
 * [**Message.cs**](ChatGPT/ChatGPT/Presentation/Message.cs): Record that represents a message sent by the user or the AI.
```

**Review by agneszitte — APPROVED** [review]

**Pull request merged**